src/util/logging.cpp

```diff
@@ -33,14 +33,24 @@ QFile s_logfile;
 // Whether to break on debug assertions.
 bool s_debugAssertBreak = false;
 
+// Note:
+// you can customize this pattern by starting Mixxx with
+// QT_MESSAGE_PATTERN="%{message}" mixxx
```

Comment:
I guess the *mixxx* suffix following the pattern was unintended?

Reply:
I guess that's for running `mixxx`.